### DISCUSSION TITLE
SFRA version AgilOne Integrations ?

### DISCUSSION THREAD

**Discussion by mkumar8687** [SalesforceCommerceCloud/sfcc-ci#466] [Team Posts]
Hi guys,
I need to integrate the **AgilOne** for my client. I did not find any cartridge for the same?
If any of you know or have worked on plz help me out.

Thanks to all

**Comment by chrisgilmoresf** [comment]
Hello,

AgilOne's partnership with the B2C Link team ended in 2017, and has no cartridge on the marketplace at this time. If AgilOne is interested in becoming a partner, please let me know and I'll loop in the appropriate contacts on our end!
